src/whatsnew/1.6.rst

```diff
@@ -29,7 +29,7 @@ The :ref:`Proxy Authentication <api/auth/proxy>` handler was renamed to
 ``proxy_authentication_handler`` to follow the ``*_authentication_handler`` form
 of all other handlers. The old ``proxy_authentification_handler`` name is marked
 as deprecated and will be removed in future releases. It's strongly recommended
-to update :config:option:`httpd/authentication_handlers` option with new value
+to update ``httpd/authentication_handlers`` option with new value
```

Comment:
I had to de-reference this because the 1.6 config option no longer exists verbatim, and it would be wrong to link it to the newer option that doesn't exist back in 1.6.